keadm/cmd/keadm/app/cmd/edge/reset_others.go

```diff
@@ -51,6 +51,12 @@ func NewOtherEdgeReset() *cobra.Command {
 		Long:    resetLongDescription,
 		Example: resetExample,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if reset.PreRun != "" {
+				fmt.Printf("Execute the prescript before joining the node")
```

Comment:
```suggestion
				fmt.Printf("Execute the prescript before resetting the node")
```

Reply:
Thank you for your suggestion. I have modified it.